src/BuildScriptGenerator/ToolNameConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.Oryx.BuildScriptGenerator
+{
+    /// <summary>
+    /// Represents tool names for different platforms.
+    /// </summary>
+    public static class ToolNameConstants
```

Comment:
nit: PlatformToolNameConstants?

Reply:
Not sure which is more suitable, PlatformTooNamesConstants should be great.

Reply:
i personally like PlatformToolNames; If we want, we can shove it in a Constants namespace to make it clearer that it contains constants